### ISSUE TITLE
Serializer class improvement suggestion

### ISSUE THREAD

**Issue by hauru** [bruth/django-preserialize#16]
Hi,

first of all, thanks for the recent update. Serializer class seems useful to get the code organized better. I have some additional suggestions though that i hope make sense. 

The Serializer-level options could be perhaps implemented as class/instance attributes. This would make it possible to have a base serializer class from which defaults could be inherited. What could also be useful is for the `related` dictionary to accept Serializer class / instances as values. With these two things together it will be possible to declare and assign common settings in just one place - the base serializer  class. It makes sense in case of hooks, for example, when a common logic needs to be executed for any instance of every serializable model.

**Comment by bruth** [comment]
> The Serializer-level options could be perhaps implemented as class/instance attributes. This would make it possible to have a base serializer class from which defaults could be inherited.

Good idea. I suppose it can be layered. Class attributes, then instance attributes, then runtime arguments.

> What could also be useful is for the related dictionary to accept Serializer class / instances as values.

Yes.
